pkg/queryfrontend/roundtrip.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
+	"strings"
```

Comment:
CI failed. Can you please group the import? Thanks!

Reply:
https://github.com/Oghenebrume50/thanos/blob/fcd80a2ec730eacc29f49a1076f802704e5d632c/pkg/queryfrontend/roundtrip.go#L6-L19

@yeya24 would this be a proper way to group the imports? I am new to Go  and Thanos and I am not so clear on how to do the grouping

Reply:
Standard library imports should be grouped together and should be the top of the file. We have tooling for it to check on your development machine. Please take a look at the Makefile, you'll find them. (`goimports`)

Reply:
Thank you, I have done the corrections